src/Glpi/Controller/Helpdesk/IndexController.php

```diff
@@ -69,7 +69,7 @@ public function __invoke(Request $request): Response
         return $this->render('pages/helpdesk/index.html.twig', [
             'title' => __("Home"),
             'menu'  => ['helpdesk-home'],
-            'tiles' => $this->tiles_manager->getTiles(),
+            'tiles' => $this->tiles_manager->getTiles(Session::getCurrentProfile()),
```

Comment:
The tiles must probably filtered depending on the current entities scope. Indeed, the user may have selected a specific entity and one of the form could not be available on this specific entity. In this case, the tile should not be displayed.

Reply:
Good catch, we probably also need to check that the form itself is visible (allow list, ...).
I'll look into it.